From: Benjamin Barenblat <bbaren@google.com>
Subject: Do not propagate -Wno-... flags into pkg-config files
Forwarded: not-needed

Lintian doesn't like any -W flags in pkg-config files, even if those flags
disable warnings.

Index: abseil/CMake/AbseilHelpers.cmake
===================================================================
--- abseil.orig/CMake/AbseilHelpers.cmake
+++ abseil/CMake/AbseilHelpers.cmake
@@ -195,9 +195,6 @@ function(absl_cc_library)
         # An -Xarch_ flag implies that its successor only applies to the
         # specified platform. Such option groups are each specified in a single
         # `SHELL:`-prefixed string in the COPTS list, which we simply ignore.
-      elseif(${cflag} MATCHES "^(-Wno-|/wd)")
-        # These flags are needed to suppress warnings that might fire in our headers.
-        set(PC_CFLAGS "${PC_CFLAGS} ${cflag}")
       elseif(${cflag} MATCHES "^(-W|/w[1234eo])")
         # Don't impose our warnings on others.
       elseif(${cflag} MATCHES "^-m")
